gix-transport/src/client/blocking_io/http/mod.rs

```diff
@@ -215,13 +212,6 @@ impl Default for Options {
     }
 }
 
-/// The actual http client implementation, using curl
-#[cfg(feature = "http-client-curl")]
-pub type Impl = curl::Curl;
-/// The actual http client implementation, using reqwest
-#[cfg(feature = "http-client-reqwest")]
-pub type Impl = reqwest::Remote;
-
```

Comment:
Initially I wanted to bring `Impl` back as it's a nice way to get the implementation of what's actually implemented, but then I realised that this is quite an implementation detail and it's probably unused outside of this crate anyway.